test/controllers/user_controller_test.exs

```diff
@@ -126,6 +126,7 @@ defmodule CodeCorps.UserControllerTest do
       assert user.biography == "Just a test user"
     end
 
+    @tag :s3
```

Comment:
Is there a more generalizable tag we could use here, like `requires_env` or something?